examples/notebooks/05_market_comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "m0DaRwFA7VgW",
+    "outputId": "5655adad-5b7a-4fe3-9067-6b502a06136b",
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install assume-framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "IIw_QIE3pY34"
+   },
+   "source": [
+    "And easy like this we have ASSUME installed. Now we can let it run. Please note though that we cannot use the functionalities tied to docker and, hence, cannot access the predefined dashboards in colab. For this please install docker and ASSUME on your personal machine.\n",
+    "\n",
+    "To run the examples, we still need some packages imports and configure a database server URI - you can adjust this if needed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "3eVM60Qx8SC0",
+    "outputId": "20434515-6e65-4d34-d44d-8c4529a46ece"
+   },
+   "outputs": [],
+   "source": [
+    "from functools import partial\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from sqlalchemy import create_engine\n",
+    "\n",
+    "from assume import World\n",
+    "from assume.scenario.loader_csv import load_scenario_folder\n",
+    "\n",
+    "# make sure that you have a database server up and running\n",
+    "DB_URI = \"postgresql://assume:assume@localhost:5432/assume\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can run the first scenario which does only include the energy-only market, we call this \"eom_case\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "world = World(database_uri=DB_URI)\n",
+    "load_scenario_folder(\n",
+    "    world,\n",
+    "    inputs_path=\"examples/inputs\",\n",
+    "    scenario=\"example_02c\",\n",
+    "    study_case=\"eom_case\",\n",
+    ")\n",
+    "world.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can run the second case which also includes the long term market as can be seen in the config.yaml of it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "world = World(database_uri=DB_URI)\n",
+    "load_scenario_folder(\n",
+    "    world,\n",
+    "    inputs_path=\"examples/inputs\",\n",
+    "    scenario=\"example_02c\",\n",
+    "    study_case=\"ltm_case\",\n",
+    ")\n",
+    "world.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zMyZhaNM7NRP"
+   },
+   "source": [
+    "## 3. Visualize the results\n",
```

Comment:
really nice with the visualization. Could you please also here change it to a local db and have the timescale as comment for local use? We need to make sure, that all these notebooks work when started from collab